test/View/Console/DefaultRenderingStrategyTest.php

```diff
@@ -90,4 +90,35 @@ public function testIgnoresNonConsoleModelNotContainingResultKeyWhenObtainingRes
         $content = $response->getContent();
         $this->assertNotContains('Page not found', $content);
     }
+
+    public function testIgnoresNonModel()
+    {
+        $console = $this->getMock('Zend\Console\Adapter\AbstractAdapter');
+        $console
+            ->expects($this->any())
+            ->method('encodeText')
+            ->willReturnArgument(0)
+        ;
+
+        //Register console service
+        $sm = new ServiceManager();
+        $sm->setService('console', $console);
+
+        /** @var \PHPUnit_Framework_MockObject_MockObject|\Zend\Mvc\ApplicationInterface $mockApplication */
+        $mockApplication = $this->getMock('Zend\Mvc\ApplicationInterface');
+        $mockApplication
+            ->expects($this->any())
+            ->method('getServiceManager')
+            ->willReturn($sm)
+        ;
```

Comment:
No strong feelings on it, I guess we'll eventually add the rule later on, if this keeps popping up